.github/workflows/test.yml

```diff
@@ -88,3 +88,81 @@ jobs:
               xvfb-run python make.py check &&
               echo 'Finished successfully! :)'
             "
+
+  test-pios:
+    name: Test Raspbian ${{ matrix.docker-tag }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        docker-tag: ['stretch-2018-03-13', 'buster-2021-03-25']
+      fail-fast: false
+    services:
+      rpios:
+        # Custom made images for Mu: https://github.com/carlosperate/docker-qemu-rpi-os
+        image: ghcr.io/carlosperate/qemu-rpi-os-lite:${{ matrix.docker-tag }}-mu
+        ports: ["5022:5022"]
+    steps:
+      # This delay is a bit hacky, but can't find a way to signal when OS is ready
+      - name: Wait 2m30s for the docker image to start up QEMU and Raspberry Pi OS
+        run: sleep 150
```

Comment:
This step is a bit hacky, but will look into it in the next revision (when adding tests for `pip install mu` in the system Python).